neuralmonkey/runners/rnn_runner.py

```diff
@@ -42,13 +42,14 @@
 
 
 def _n_best_indices(scores: np.ndarray, beam_size: int) -> np.ndarray:
+    """Indices of the n-best list based on provided scores."""
     if scores.shape[0] <= beam_size:
         unsorted_n_best_indices = np.arange(scores.shape[0])
     else:
         unsorted_n_best_indices = np.argpartition(
             -scores, beam_size)[:beam_size]
     n_best_indices = unsorted_n_best_indices[
-        np.argsort(scores[unsorted_n_best_indices])]
+        np.argsort(-scores[unsorted_n_best_indices])]
```

Comment:
Blbý je, že to stejně furt nefunguje. Až si udělám čas, tak asi napíšu unit testy na ty jednotlivý funkce.